### DISCUSSION TITLE
can be installed on windows without admin rights?

### DISCUSSION THREAD

**Discussion by AlaEddineBoughanmmi** [elvirbrk/NoteHighlight2016#140] [Q&A]
is there a way to install the package using source code  on windows without admin rights  ?

**Comment by gamalielcabria** [comment]
I also support this inquiry. As a post graduate student, university computers does not provide us admin rights. Installation via the .msi file requires admin rights. Is there a protocol to install it without it? even if its a manual one? Thanks

**Comment by StanDaMan0505** [comment]
Did you find a way to install it without admin rights?
